packages/cursorless-engine/src/generateSpokenForm/generateSpokenForm.ts

```diff
@@ -36,6 +38,23 @@ export class SpokenFormGenerator {
     );
   }
 
+  getSpokenFormForSingleTerm<T extends SpokenFormType>(
+    type: T,
+    id: SpokenFormMapKeyTypes[T],
+  ): SpokenForm {
+    return this.componentsToSpokenForm(() => {
+      const value = this.spokenFormMap[type][
+        id as unknown as keyof SpokenFormComponentMap[T]
+      ] as SpokenFormComponent;
```

Comment:
Sometimes typescript baffles me. Could not for the life of me get it to be happy with this one

Reply:
That type is quite complex, but yeah I couldn't get it to work either.